bcolz/toplevel.py

```diff
@@ -273,22 +288,28 @@ def fill(shape, dflt=None, dtype=np.float, **kwargs):
     # Create the container
     expectedlen = kwargs.pop("expectedlen", length)
     if dtype.kind == "V" and dtype.shape == ():
-        raise ValueError("fill does not support ctables objects")
-    obj = bcolz.carray([], dtype=dtype, dflt=dflt, expectedlen=expectedlen,
+        list_carrays = []
+        base_rootdir = kwargs.pop('rootdir', None)
+        if base_rootdir is not None:
+            os.mkdir(base_rootdir)
+        for name,col_dype in dtype.descr:
```

Comment:
PEP8: need space between `name` and `col_dype`.  Also, `col_dype` -> `col_dtype`.